docs/intro/tutorial.rst

```diff
@@ -564,7 +564,7 @@ this time for scraping author information::
 
         def parse(self, response):
             # follow links to author pages
-            for href in response.css('.author a::attr(href)').extract():
+            for href in response.css('span a::attr(href)').extract():
```

Comment:
Hi, eliasdorneles
Of course, I agree that it is important to write code explicitly. It is my first time to contact with scrapy and web crawl. I followed the tutorial and found the original code does't work on my computer. So I slightly changed it.

Thanks for your attention!